docs/marketplace-required-csv-annotations.md

```diff
@@ -5,25 +5,56 @@ An operator's CSV must contain the following annotations for it to be displayed
 ```yaml
 metadata:
   annotations:
-    categories: A list of comma separated categories that your operator falls under.
-    certified: The operator's certification. This field should be set to false until a process to get this value is defined at a later date.
+    capabilities: One of the following: Basic Install, Seamless Upgrades, Full Lifecycle, Deep Insights, Auto Pilot. For more information see https://www.operatorhub.io/images/capability-level-diagram.svg
+    categories: A list of comma separated list of categories from the values below. If not set, this will be set to "Other" in the UI.
+    containerImage: The repository that hosts the operator image. The format should match ${REGISTRYHOST}/${USERNAME}/${NAME}:${TAG}
+    createdAt: The date that the operator was created. The format should match yyyy-mm-ddThh:mm:ssZ
+    support: The maintainer of the operator. This value should be a name, not an email.
+    repository: (Optional) a URL to a source code repository of the Operator, intended for community Operators to direct users where to file issues / bug
+    alm-examples: A string of a JSON list of example CRs for the operator's CRDs. These should be able to work without modification, unless otherwise stated in the operator description.
```

Comment:
AFAIK the scorecard tries to use these directly so that will be an issue if it needs changes for use. But we should confirm with the SDK team before making this statement.